R/derive_param_computed.R

```diff
@@ -117,13 +117,18 @@
 #' @param keep_nas Keep observations with `NA`s
 #'
 #'   If the argument is set to `TRUE`, observations are added even if some of
-#'   the values contributing to the computed value are `NA`.
+#'   the values contributing to the computed value are `NA` (see Example 1b).
```

Comment:
Be so cool if we could hyperlink to this example

Reply:
Yes, linking to examples would be really cool.

Unfortunately, I'm not aware of an easy way to do this. We discussed creating a roclet for such improvements in https://github.com/pharmaverse/admiral/issues/1590 but it was never implemented and I don't think we have resources for this.